src/components/views/right_panel/UserInfo.tsx

```diff
@@ -165,10 +165,17 @@ function useHasCrossSigningKeys(
     }, [cli, member, canVerify]);
 }
 
-export function DeviceItem({ userId, device }: { userId: string; device: IDevice }): JSX.Element {
+export function DeviceItem({
+    userId,
+    device,
+    isUserVerified,
```

Comment:
it would be nice to document (here and in `DevicesSection`) that this is `false` initially, while we look up the verification status from the store.